### ISSUE TITLE
No longer can be used now.

### ISSUE THREAD

**Issue by shadowdragon1123** [cheems/Mega-to-Google-Drive#32]
Now the google block my account's colab access permissions.

**Comment by NabarunKar** [comment]
Can you provide more details or show an image of the exact issue you are facing? I think you might have exceeded the usage limit or violated their ToS.

**Comment by shadowdragon1123** [comment]
![L{(0PNF~}D(B}%EF33{WY03](https://github.com/user-attachments/assets/037eda46-991f-4d7a-99b7-ed4d5af73296)

**Comment by NabarunKar** [comment]
Yeah, you definitely triggered their filter which detects abuse and got blocked. This has nothing to do with this repository.

**Comment by shadowdragon1123** [comment]
Is this means that I can register a new account and can continue use it again?

**Comment by shadowdragon1123** [comment]
> Yeah, you definitely triggered their filter which detects abuse and got blocked. This has nothing to do with this repository.

But I have another account, it only use to transfer 200 gb last month.
It is first time use, but also is blocked now.

**Comment by Apostatethings** [comment]
Same here, had two of my accounts blocked from accessing the runtime

**Comment by Apostatethings** [comment]
I think they changed their policies on whats allowed and what isn't.
![Screenshot 2024-11-24 035747](https://github.com/user-attachments/assets/4c66d689-2c61-4cfb-b66e-3df263685cb8)
